packages/ui/src/components/navbar-project-chooser.tsx

```diff
@@ -36,7 +36,7 @@ function Root({ logo }: ProjectProps) {
         handleChange={openSearchDialog}
       />
       <Dialog open={isSearchDialogOpen} onOpenChange={closeSearchDialog}>
-        <DialogContent className="h-[600px] max-w-[800px] border-border bg-primary-background">
+        <DialogContent className="h-[600px] max-w-[800px] border-border bg-background-1">
```

Comment:
Wouldn't this need a XD review as well, since this change does not only pertain to light mode but would reflect in dark mode as well?

Reply:
Yes @vardanbansal-harness , we have this variant added in light theme as well.